### PR TITLE
feat: calculate r-squared(Coefficient of Determination) problem no. 70

### PR DESCRIPTION
**Pull request by rittik9**
All the test cases are according to sklearn r2_score implementation.

### CODE REVIEW

File: Problems/70_r-squared/learn.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+<p>The R-squared value is calculated using the following formula:</p>
+
+\[
+R^2 = 1 - \frac{\text{SSR}}{\text{SST}}
+\]
+
+<p>Where:</p>
+<ul>
+    <li><strong>SSR \((\)Sum of Squared Residuals\()\):</strong> The sum of the squares of the differences between the actual values and the predicted values.</li>
```
Comment (moe18) [inline Problems/70_r-squared/learn.html:15]: I would show the equation for SSR

File: Problems/70_r-squared/learn.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <li><strong>SST \((\)Total Sum of Squares\()\):</strong> The sum of the squares of the differences between the actual values and the mean of the actual values.</li>
```
Comment (moe18) [inline Problems/70_r-squared/learn.html:16]: I would show the equation for SST

**Review by moe18 — APPROVED** [review]
looks good, had a small comment

File: Problems/70_r-squared/learn.html (hunk 1 above)
Comment (rittik9) [reply Problems/70_r-squared/learn.html:9]: done

**Review by moe18 — APPROVED** [review]

**Pull request merged**